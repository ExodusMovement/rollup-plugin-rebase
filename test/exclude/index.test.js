import { bundle, clean, list, read } from "../util"

const root = __dirname

test.only("Exclude", async () => {
  await bundle(root, "index.js", "output/index.js", { exclude: ["**/*.png"] })

  expect(await list(root, "output")).toMatchSnapshot()
  expect(await read(root, "output/index.js")).toMatchSnapshot()
  expect(await read(root, "output/cGxRTWKw.css")).toMatchSnapshot()

  await clean(root, "output")
})
