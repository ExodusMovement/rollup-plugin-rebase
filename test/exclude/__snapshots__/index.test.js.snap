// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exclude 1`] = `
"HDWIGpiP.md
index.js
vfhBOfeG.css
vfhBOfeG.css.map"
`;

exports[`Exclude 2`] = `
"import content from './HDWIGpiP.md';
import css from './vfhBOfeG.css';

function index() {
  return content + css
}

export { index as default };
"
`;

exports[`Exclude 3`] = `
"body {
  font-size: 20px;
  background: blue;
  color: white;
}

/*# sourceMappingURL=vfhBOfeG.css.map */"
`;
